### PR TITLE
Fix types in the doc

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: contribs/gmf/src/lidarprofile/Measure.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * @module ngeo.lidarprofile.Measure
+ * @module import("ngeo/lidarprofile/Measure.js").default
```
Comment (gberaudo) [inline contribs/gmf/src/lidarprofile/Measure.js]: Should not replace for modules

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ GeoMapFish-specific web services. The `gmf` contrib examples are also available
 ### Run the application
 
 ```
-git clone git@github.com:camptocamp/ngeo.git
+git clone git@github.com:camptocamp/import("ngeo/git.js").default
```
Comment (fredj) [inline README.md]: this should not be changed

File: buildtools/deploy.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@ fi
 
 # Cloning gh-pages into a local temporary directory
 TMP=".build/ngeo-${GITHUB_USERNAME}-gh-pages"
-git clone --single-branch --branch gh-pages git@github.com:${GITHUB_USERNAME}/ngeo.git ${TMP}
+git clone --single-branch --branch gh-pages git@github.com:${GITHUB_USERNAME}/import("ngeo/git.js").default ${TMP}
```
Comment (fredj) [inline buildtools/deploy.sh]: this should not be changed

File: README.md (hunk 2 above)
Comment (sbrunner) [reply README.md]: Tanks :-)

**Pull request merged**